lib/generators.js

```diff
@@ -30,9 +30,9 @@ function generateRequestFile(schema, settings) {
 
   // For each CRUD function, call the corresponding snippet function on each table in schema
   // and append to fileContents.
-  ['create', 'read', 'update', 'delete'].forEach(fn => {
+  ['create', 'read', 'update', 'delete'].forEach((fn) => {
```

Comment:
i’m seeing some differences in our eslint configs here LOL

Reply:
wait I feel like this got changed on your side in #34 😂